lib/features/reps_core/reps_core.module

```diff
@@ -839,7 +839,22 @@ function reps_core_menu_local_tasks_alter(&$data, $router_item, $root_path) {
  * Implements hook_form_FORM_ID_alter().
  */
 function reps_core_form_page_node_form_alter(&$form, &$form_state, $form_id) {
-   if (!isset($form['field_ne_body'])) {
-     $form['body']['#weight'] = -2;
-   }
+  if (!isset($form['field_ne_body'])) {
+    $form['body']['#weight'] = -2;
+  }
+}
+
+/**
+ * Implements hook_views_pre_render().
+ */
+function reps_core_views_pre_render(&$view) {
```

Comment:
why are you not using the permissions / role in the view itself?

Reply:
@steeff, 
Those views are generated with nodequeue and are not exported into feature. ID are different depending the country. So I've implemented this hook to avoid making the changes manually in all websites.

Reply:
ok
did you try hook_views_default_views_alter() ? but not sure these kind of views are recognize as such.... but try and let me know